src/fshtypes/RuleSet.ts

```diff
@@ -19,4 +20,8 @@ export class RuleSet extends FshEntity {
   get id() {
     return this.name;
   }
+
+  ruleIsAllowed(rule: Rule) {
+    return !(rule instanceof InsertRule);
+  }
```

Comment:
I'm confused.  Don't we allow `InsertRule` in `RuleSet`s?

Reply:
Well we do, but when you are testing if a `ruleIsAllowed`, that test is after everything has been expanded. So I guess this function should have a better name than `ruleIsAllowed`, since it is really `ruleIsAllowedAfterExpandingInsertRules`. But that is too long. Maybe `ruleApplies`? That is still not totally clear, but maybe more clear?